src/App.jsx

```diff
@@ -13,15 +13,15 @@ export default function App() {
       <ProvideAuth>
         <Router>
           <Switch>
-            <Route exact path="/">
```

Comment:
All the exact does here is check to make sure the route is exact to path you will need this on a more complex application..